src/repositories/PriceDataRepository.ts

```diff
@@ -50,4 +56,35 @@ export class PriceDataRepository {
       this.logger.error(`couldn't perform bulkWrite for PriceData: ${error}`);
     }
   }
+
+  async latest(limit = 150): Promise<PriceDataModel[]> {
+    return getModelForClass(PriceDataModel).find().sort({timestamp: -1}).limit(limit).exec();
+  }
+
+  async latestSymbols(): Promise<string[]> {
+    const symbols = await getModelForClass(PriceDataModel).find({}, {feedBase: 1, feedQuote: 1}).limit(1000).exec();
+    const unique: Record<string, number> = {};
+
+    // TODO: this logic is for me a bit complicated to understand
+    // maybe a function can help to do thar more clear
+    //
+    // I can just understand that in case there are these feeds:
+    // SYMBOL1
+    // SYMBOL2
+    // SYMBOL1
+    // it orders them like SYMBOL1, SYMBOL2 because SYMBOL1 appears more often than SYMBOL2
+    symbols.forEach((s) => {
+      const symbol = s.feedBase + '-' + s.feedQuote;
+      unique[symbol] = unique[symbol] ? unique[symbol] + 1 : 1;
+    });
+
+    return Object.keys(unique).sort((a, b) => {
+      return unique[a] == unique[b] ? (a < b ? -1 : 1) : unique[b] - unique[a];
+    });
```

Comment:
@DZariusz let me know what do you think.

Reply:
why you removed `symbol: string, limit = 150` from arguments? and why previous implementation will not work?

Reply:
Do you mean the function below? `lastestPrice` ? 
If this is the case, these arguments are there